getting_started/data_backup.md

```diff
@@ -105,7 +105,7 @@ We have provided additional instructions for [Google Drive](#google-drive) and [
 
 After completing [steps 1 - 3 above](#configuring-rclone-remotes): you will be prompted to enter a [`client_id`](https://rclone.org/drive/#making-your-own-client-id)
 
-1. When prompted from a [`client_id`](https://rclone.org/drive/#making-your-own-client-id), follow the instruction on the linked page.
+1. When prompted for a [`client_id`](https://rclone.org/drive/#making-your-own-client-id), follow the instruction on the linked page.
```

Comment:
let's say follow the instructions here, and link here to the clone link. It's just a little unwieldy as is.

Reply:
Done.